### PR TITLE
Add a switch to the email templates for brand styles [MAILPOET-5193]

### PR DESCRIPTION
**Pull request by samnajian**
## Description

This PR adds the boxes to choose between the brand and theme styles, per the visuals provided in the ticket.

## Code review notes

- There are some to-do items added which are hard-coded values that should be replaced in the following tickets.
- Tried converting settings.js to ts, but that started to take longer than expected, and since we're anyways rendering a React component for this new feature inside it, thought I'll leave the conversion to another time.

## Linked tickets

[MAILPOET-5193]

[MAILPOET-5193]: https://mailpoet.atlassian.net/browse/MAILPOET-5193?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/assets/js/src/newsletter_editor/blocks/sidebar/brandStyles.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { __ } from '@wordpress/i18n';
+
+export function BrandStyles() {
+  return (
+    <>
+      <h4>
+        {__('Choose a style')} <i className="mailpoet_info" />
```
Comment (costasovo) [inline mailpoet/assets/js/src/newsletter_editor/blocks/sidebar/brandStyles.tsx]: I think you need to add the domain parameter here.

`__('Choose a style', 'mailpoet')`

**Review by costasovo — CHANGES_REQUESTED** [review]
Looks good! I found just a little issue in a translation string.

File: mailpoet/assets/js/src/newsletter_editor/blocks/sidebar/brandStyles.tsx (hunk 1 above)
Comment (samnajian) [reply mailpoet/assets/js/src/newsletter_editor/blocks/sidebar/brandStyles.tsx]: Fixed in 03edc465caac8e494a895aa5370d34c414534dee

**Review by costasovo — DISMISSED** [review]
👍

**Comment by costasovo** [comment]
@samnajian I already approved the PR, but when I returned to my browser, I noticed there was a react warning coming from the new code.

I guess this is something we should address.

Screenshot 2023-04-11 at 13 31 08

**Comment by samnajian** [comment]
> @samnajian I already approved the PR, but when I returned to my browser, I noticed there was a react warning coming from the new code.
> 
> I guess this is something we should address.
> 
> Screenshot 2023-04-11 at 13 31 08

Fixed in f6ae74d3cf4224ec72178e16d88356256fd25507

**Review by costasovo — APPROVED** [review]
Thanks for fixing the warning!

**Review by Aschepikov — APPROVED** [review]
GJ!

**Pull request merged**